From: Kei Tokunaga <ktokunag@redhat.com>
Subject: [RHEL5.1 PATCH 13/21] Fix for irq_desc() missing in new upstream
Date: Thu, 07 Jun 2007 03:42:56 -0400
Bugzilla: 242137
Message-Id: <4667B700.20700@redhat.com>
Changelog: [xen] ia64: Fix for irq_desc() missing in new upstream


bz242137
https://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=242137

Backport of cset#13859.

In upstream, irq_descp() have already been replaced
with irq_desc().  Need to replace irq_descp() used
in a xen function as well.

Thanks,
Kei


rh bug 242137

# HG changeset patch
# User awilliam@xenbuild2.aw
# Date 1170027019 25200
# Node ID fe3c36600d14d157e0fbbd5ab635e625e1031acf
# Parent  8b37c7bba7e12d306e720718fe8e1e6d6dee28eb
[IA64] Fix for irq_desc() missing in new upstream

Signed-off-by: Alex Williamson <alex.williamson@hp.com>


---

 linux-2.6.18-21.el5-gerd-order-kei/arch/ia64/kernel/irq_ia64.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff -puN arch/ia64/kernel/irq_ia64.c~13859-IA64_Fix_for_irq_desc_missing_in_new_upstream arch/ia64/kernel/irq_ia64.c
--- linux-2.6.18-21.el5-gerd-order/arch/ia64/kernel/irq_ia64.c~13859-IA64_Fix_for_irq_desc_missing_in_new_upstream	2007-06-07 02:44:32.000000000 -0400
+++ linux-2.6.18-21.el5-gerd-order-kei/arch/ia64/kernel/irq_ia64.c	2007-06-07 02:44:32.000000000 -0400
@@ -388,7 +388,7 @@ xen_register_percpu_irq (unsigned int ve
 			 * mark the interrupt for migrations and trigger it
 			 * on cpu hotplug.
 			 */
-			desc = irq_descp(irq);
+			desc = irq_desc + irq;
 			desc->status |= IRQ_PER_CPU;
 		}
 	} 

_

